test/box-upgradability.test.ts

```diff
@@ -47,11 +45,7 @@ describe('Box Upgrade contract', () => {
 
     beforeEach(async () => {
         box = await deployContractWithProxy<Box>('Box')
-        upgradedListener = new EventListener<ActualUpgradedEvent>(
-            box,
-            'Upgraded',
-            (event) => upgradedEvent(event)
-        )
+        upgradedListener = wrapContractEvent(box, 'Upgraded').newListener()
```

Comment:
suggestion: To keep the tests nice and readable.

Change method name & wrapping from:
```
upgradedListener = wrapContractEvent(box, 'Upgraded').newListener()
```
to 
```
upgradedListener = contractEventListener(box, 'Upgraded')
```

Reply:
I prefer chaining style because it allows to keep an intermediate result, e.g.
`const eventUpgraded = wrapContractEvent(box, 'Upgraded')
...
upgradedListener = eventUpgraded.newListener()
`

but as performance optimization is not a case here - ok, lets do as isolated functions ;)